src/components/Screens/OurApps.js

```diff
@@ -7,7 +7,35 @@ import "src/styles/Screens/OurApps.css";
 const settings = {
   wrapAround: true,
   speed: 500,
-  slidesToShow: 1
+  slidesToShow: 1,
+  decorators: [{
+    component: React.createClass({
```

Comment:
Refactor this inline component into separate entity file under`src/components/` and give it a name.